core/communication/bubblewrap/src/main/javascript/index.ts

```diff
@@ -218,50 +229,20 @@ export class Bubblewrap {
 
     /**
      * Constructs a new, independent [[Bubblewrap]] instance with additional registered classes.
+     * TODO: Find out why this creates an additional object
      *
      * This method throws an exception if there is a duplicate class identifier.
      */
     addClasses(more: Classes): Bubblewrap {
         const thisKeys = Object.keys(this.classes);
-        const thatKeys = Object.keys(more);
-        for (const thisKey of thisKeys)
-            if (thatKeys.includes(thisKey)) throw new Error(`Duplicate identifier ${thisKey}`);
+        const thatsKeys = Object.keys(more);
+        for (const aKey of thatsKeys)
+            if (thisKeys.includes(aKey)) throw new Error(`Duplicate identifier ${aKey}`);
```

Comment:
Works for me I guess - though I find the original naming easier to follow FWIW. `thatKeys` was consistently named to `thisKeys`, you switched around which array of keys you're checking first which doesn't matter, and `aKey` makes me try to find `bKey` :D (`aSomething` is pretty uncommon in the JS world, it's normally just `something`)

Reply:
Right on that...